application/use_cases/stock_locations/delete_stock_location/idelete_stock_location_output_port.py

```diff
@@ -0,0 +1,16 @@
+from abc import ABC, abstractmethod
+
+from clapy import IOutputPort
+
+from domain.entities.base_entity import EntityID
+
+
+class IDeleteStockLocationOutputPort(IOutputPort, ABC):
+
+    @abstractmethod
+    async def stock_location_deleted_async(self, stock_location_id: EntityID) -> None:
```

Comment:
naming should be present_xyz

also id not relevant anymore?

Reply:
nah not relevant, removed